Add recipe for aiapy

wtbarnes opened the pull request:
<!--
Thank you very much for putting in this recipe PR!

This repository is very active, so if you need help with
a PR or once it's ready for review, please let the right people know.
There are language-specific teams for reviewing recipes.

Currently available teams are:
- python `@conda-forge/help-python`
- python/c hybrid `@conda-forge/help-python-c`
- r `@conda-forge/help-r`
- java `@conda-forge/help-java`
- nodejs `@conda-forge/help-nodejs`
- c/c++ `@conda-forge/help-c-cpp`
- perl `@conda-forge/help-perl`
- Julia `@conda-forge/help-julia`
- ruby `@conda-forge/help-ruby`

If your PR doesn't fall into those categories please contact
the full review team `@conda-forge/staged-recipes`.

Due to GitHub limitations first time contributors to conda-forge are unable
to ping these teams. You can [ping the team](https://conda-forge.org/docs/maintainer/infrastructure.html#conda-forge-admin-please-ping-team) using a special command in
a comment on the PR to get the attention of the `staged-recipes` team. You can
also consider asking on our [Gitter channel](https://gitter.im/conda-forge/conda-forge.github.io)
or on our [Keybase chat](https://keybase.io/team/condaforge.chat)
if your recipe isn't reviewed promptly.
-->

Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/aiapy```) and found some lint.

Here's what I've got...


For **recipes/aiapy**:

 * ``requirements: run: sunpy>=2.0`` must contain a space between the name and the pin, i.e. ``sunpy >=2.0``

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/aiapy```) and found it was in an excellent condition.

wtbarnes commented:
This is blocked by #13355

dopplershift commented on recipes/aiapy/meta.yaml:
```diff
@@ -0,0 +1,50 @@
+    - python
+    - setuptools_scm
+  run:
+    - importlib_resources  # [py<37]
```
It's safe to just have this dependency on `importlib_resources` without the selector. Yes, it's redundant with the standard library package on Python >= 3.8, but then you can make this `noarch:python`, which is a big win.

wtbarnes replied:
```suggestion
    - importlib_resources
```

dopplershift commented on recipes/aiapy/meta.yaml:
```diff
@@ -0,0 +1,50 @@
+requirements:
+  host:
+    - pip
+    - python
```
When making noarch, make this `python >=3.6`.

wtbarnes replied:
```suggestion
    - python >=3.6
```

dopplershift commented on recipes/aiapy/meta.yaml:
```diff
@@ -0,0 +1,50 @@
+    - setuptools_scm
+  run:
+    - importlib_resources  # [py<37]
+    - python
```
`python >=3.6` here too

wtbarnes replied:
```suggestion
    - python >=3.6
```

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/aiapy```) and found some lint.

Here's what I've got...


For **recipes/aiapy**:

 * Non noarch packages should have python requirement without any version constraints.
 * Non noarch packages should have python requirement without any version constraints.

wtbarnes commented on recipes/aiapy/meta.yaml:
```diff
@@ -0,0 +1,50 @@
+
+build:
+  number: 0
+  skip: true   # [py2k]
```
```suggestion
  skip: true   # [py2k]
  noarch: python
```

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I wanted to let you know that I linted all conda-recipes in your PR (```recipes/aiapy```) and found some lint.

Here's what I've got...


For **recipes/aiapy**:

 * `noarch` packages can't have selectors. If the selectors are necessary, please remove `noarch: python`.

wtbarnes commented on recipes/aiapy/meta.yaml:
```diff
@@ -0,0 +1,51 @@
+
+build:
+  number: 0
+  skip: true   # [py2k]
```
```suggestion
```

conda-forge-linter commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipes/aiapy```) and found it was in an excellent condition.

wtbarnes commented:
Thanks for the feedback @dopplershift! I think I've addressed all of your comments. I had to remove the `skip: true` and add a `noarch: python` to make the linter happy as well.
